showcase.html

```diff
@@ -607,20 +417,6 @@ <h6>T Anime</h6>
                 <p>Anime Tracker</p>
             </div>
         </a>
-        <a class="item" href="https://github.com/sinkaroid/sinkaroid">
```

Comment:
This one seems to be working

Reply:
The website just leads to someone's personal profile. It's not a tool that anyone else can use which was my criteria. I guess it technically does work if you scroll to the very bottom and it shows his favorite anime which is gathered using Jikan.

Reply:
Hmm, you're right.